hardware/tests/test_benchmark_disk.py

```diff
@@ -52,12 +52,6 @@ def setUp(self):
         self.hw_data = [('disk', 'fake-disk', 'size', '10'),
                         ('disk', 'fake-disk2', 'size', '15')]
 
-    def test_disk_perf_text(self, mock_popen):
```

Comment:
ditto

Reply:
because the output of commands in Python 3.x is bytes, not pure text, so we don't need this, we already have the correspondent bytes test